problems/getCountriesSortedByPopulation.js

```diff
@@ -7,8 +7,13 @@
 * 
 */
 
-function getCountriesSortedByPopulation() {
-
+function getCountriesSortedByPopulation(array) {
+    let output = "";
+  return array.sort((country1, country2) => {
+    return country2.population - country1.population;
+    
+  });
+  return output
```

Comment:
You never made any changes to output... but also already returned on line 12. Instead you need to chain a map call so that you can return the correct property from the array of objects.